johngorithm/ride-my-way-react-frontend
# #61302332 log user out of the application

**What does this PR do?**
Makes it possible for users to sign out of the applications

**Description of tasks to be completed**
- create action creator function for this feature
- clear `localStorage`
- redirect a user to the landing page on successful signout

**How can this be tested**
- clone repository
- checkout `fft-user-can-logout-161302332` branch
- install dependencies by running `npm install` on the command-line
- run `npm start`
- go to `localhost:4000/login` to or `localhost:4000/register` to signup
- use the **logout** button from the dropdown that will appear when your username is `clicked`

**Associated Pivotal Tracker story**
[Delivers #161302332]
